pkg/storage/utils/decomposedfs/upload/upload.go

```diff
@@ -261,7 +263,7 @@ func (upload *Upload) FinishUpload(_ context.Context) error {
 
 	n, err := CreateNodeForUpload(upload, attrs)
 	if err != nil {
-		Cleanup(upload, true, false)
+		Cleanup(upload, true, false, true)
```

Comment:
postprocessing has not started, yet, so we might change this to 
```suggestion
		Cleanup(upload, true, false, false)
```

opinions?